### PR TITLE
integrations: Add notes for OpenShift.

### PR DESCRIPTION
**Pull request by YagoGG**
Some information about a possible integration with [OpenShift](https://www.openshift.com/) PaaS.

Screenshots aren't included because there isn't an integration in Slack for this yet.

### CODE REVIEW

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot*

@YagoGG, it looks like you've already signed the Dropbox CLA. Thanks!

File: webhook-integrations/openshift/notes.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# OpenShift
+
+[OpenShift](https://www.openshift.com/) is a PaaS (Platform-as-a-Service), quite similar to Heroku, that allows deploying and scalating cloud applications easily.
```
Comment (umairwaheed) [inline webhook-integrations/openshift/notes.md]: Typo in 'scalating'.

**Comment by umairwaheed** [comment]
@YagoGG, the other requirements of the task, e.g. screenshots, doesn't apply to this?

**Comment by YagoGG** [comment]
@umairwaheed Typo fixed! I also made lines shorter.

They don't apply either because it's an integration like Git (it's a script run from OpenShift, that we'll have to create later). 
Thus, we'll be the ones deciding what the payload is, and there is no interface to set this up that can screenshot.

**Comment by umairwaheed** [comment]
@YagoGG, hmm, I see. I also could not find the OpenShift integration in Slack's app directory.

**Comment by umairwaheed** [comment]
OK, merging this because this integration doesn't exist in Slack's app directory and I guess this
>Make a brief note in a file called notes.md (.md is the extension used by markdown format. It can be edited in any text editor e.g. notepad) about any messages you think are possible, but which you are unable to generate (e.g. because they are only available to paying customers).

applies here.

Nice job, @YagoGG.

**Pull request merged**